<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication2Class</class>
 <widget class="QMainWindow" name="QtGuiApplication2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>网卡信息：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="NetWorkcomboBox">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BtnGetNetWork">
          <property name="text">
           <string>获取网卡</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>650</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>抓包结果</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>0</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <column>
            <property name="text">
             <string>源IP</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>源MAC</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>目标IP</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>目标MAC</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>协议</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>包长</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>源端口</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>目的端口</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>数据窗口</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>16进制显示</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QTextEdit" name="textEditData"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>字符串表示</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QTextEdit" name="textEdit"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>协议分析</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QTextEdit" name="textEditProtocol"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>网卡信息</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>网卡名字 ：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LineEditNetworkName"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>描   述  ：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LineEditDecription"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>IP  地址 ：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LineEditIPAddtress"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>子网掩码 ：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LineEditNetworkMask"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>过滤设置</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>协议选择：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxprotocol">
             <item>
              <property name="text">
               <string>无</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TCP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UDP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ICMP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ARP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UNKNOW</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>源IP    ：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_SrcIP"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>源MAC   ：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditSrcMAC"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>源端口  ：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditSrcPort"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>目标IP  ：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDesIP"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>目标MAC ：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDesMAC"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>目标端口：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDesPort"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="BtnApply">
             <property name="text">
              <string>应用</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>控制台</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="BtnStart">
           <property name="text">
            <string>启动抓包</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnStop">
           <property name="text">
            <string>停止抓包</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication2.qrc"/>
 </resources>
 <connections/>
</ui>
